src/dos/dos_files.cpp

```diff
@@ -932,7 +961,7 @@ uint8_t FCB_Parsename(uint16_t seg,uint16_t offset,uint8_t parser ,char *string,
 	if(!isvalid(string[0])) goto savefcb;
 
 	hasname = true;
-	fill = ' ';
+	assert(fill == ' ');
```

Comment:
Change in functionality? Is this safe?

Reply:
I'll revert this; it's a PVS studio fix (the `fill` variable is assigned further up as an identical value, with nothing using it in-between, so this is more like a "check our assumptions" assert).

It seems like sometimes refactoring lets PVS studio "go further" and generates seemingly unrelated issues.  None the less - I'll switch back and see what happens (would rather avoid these unrelated changes anyway!).